cfme/tests/services/test_dynamicdd_dialogelement.py

```diff
@@ -265,29 +266,45 @@ def test_dynamic_field_on_textbox(request, appliance, import_datastore, import_d
     wait_for(lambda: view.fields("name_validation").read() == 'indice_1', timeout=7)
 
 
-@pytest.mark.manual
+@pytest.mark.customer_scenario
+@pytest.mark.meta(automates=[1559999])
 @pytest.mark.tier(3)
-def test_when_clicking_refresh_for_text_field_2_only_text_field_2_should_refreshed():
+@pytest.mark.parametrize("import_data", [DatastoreImport("bz_1559999.zip", "bz_1559999", None)],
+                         ids=["datastore"])
+@pytest.mark.parametrize("file_name", ["bz_1559999.yml"], ids=["refresh_dialog"])
+def test_dialog_dynamic_field_refresh_in_log(appliance, import_datastore, import_data, file_name,
+                                             generic_catalog_item_with_imported_dialog):
     """
+    Bugzilla:
+        1559999
+
     Polarion:
         assignee: nansari
+        startsin: 5.10
         casecomponent: Services
-        testtype: functional
-        initialEstimate: 1/4h
-        startsin: 5.9
-        tags: service
+        initialEstimate: 1/16h
         testSteps:
-            1.  create a dialog with two text fields with no refresh relation between them
-                showing their refresh buttons.
-            2. associate each to a different method that just logs "Refreshing X"
-            3. associating the dialog to a catalog item
-            4. tail -f log/automation.log | grep "Refreshing"
-            5. load the dialog
-            6. click "refresh" for field 2
-    Bugzilla:
-        1559999
+            1. Import Datastore and dialog
+            2. Add service catalog with above created dialog
+            3. Navigate to order page of service
+            4. In service Order page
+            5. Click on "refresh" for field 2
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5. Only text field2 should be refreshed in automation log
     """
```

Comment:
Can you please confirm!
whether it is **field2** or **RefreshField2**

Reply:
this is Field2